mirascope/core/openai/_utils/_setup_call.py

```diff
@@ -121,6 +124,8 @@ def setup_call(
                 UserWarning,
             )
         call_kwargs["tool_choice"] = "required"
+    if stream:
```

Comment:
note to self: check if stream options already exist and overwrite include usage if not present